### DISCUSSION TITLE
Reports API Access to resource is forbidden.

### DISCUSSION THREAD

**Discussion by GryphonKrinsky** [amzn/selling-partner-api-models#3519] [General]
When calling the createReport operation for the GET_ORDER_REPORT_DATA_INVOICING report type Im an getting the following error:
    "errors": [
        {
            "code": "Unauthorized",
            "message": "Access to the resource is forbidden",
            "details": ""
        }
    ] 

I know that this is not a problem with my request signature or access token since the createReport call works fine for GET_MERCHANT_LISTINGS_ALL_DATA. I have also attempted to create a RDT and use this in the x-amz-access-token header, but I still get an access denied error. How can I get access to this report? is it only available for authorised applications?

**Comment by Gopikrish25** [comment]
@GryphonKrinsky I'm facing a similar issue, can you let me know if you resolved this issue and how?

Thanks in Advance!

**Comment by mafge** [comment]
Hi @Gopikrish25,
thanks for reaching out!
Can you provide more details, e.g. which report do you want to request?
In general I would recommend to look into the following points to resolve your issue:
- Do you have all required roles to request a certain report? You can find the required role for each report in the documentation.
- Some reports are restricted to Sellers, some restricted to Vendors: Make sure that you are requesting a report that is available for the respective account type. You can find information about availability in the documentation.
- Some reports can't be requested (e.g. Settlement reports), some can only be scheduled: Make sure to check the documentation for the report you want to request.
- Make sure that your access token is valid (timeout after 60 minutes).
Hope that helps!
Best,
Marc
Selling Partner Developer Services
